src/Components/CollectionsHubRails/FeaturedCollectionsRails/__tests__/FeaturedCollectionsRails.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { CollectionHubFixture } from "Apps/__tests__/Fixtures/Collections"
+import { ArrowButton } from "Components/v2/Carousel"
+import { mount } from "enzyme"
+import "jest-styled-components"
+import React from "react"
+import {
+  FeaturedCollectionEntity,
+  FeaturedCollectionsRails,
+  FeaturedImage,
+} from "../index"
+
+describe("FeaturedCollectionsRails", () => {
+  let props
+
+  beforeEach(() => {
+    props = {
+      collectionGroup: CollectionHubFixture.linkedCollections[1],
+    }
+  })
+
+  const memberData = () => {
+    return {
+      description:
+        "<p>From SpongeBob SquarePants to Snoopy, many beloved childhood cartoons have made an impact on the history of art.</p>",
+      price_guidance: 60,
+      slug: "art-inspired-by-cartoons",
+      thumbnail: "http://files.artsy.net/images/cartoons_thumbnail.png",
+      title: "Art Inspired by Cartoons",
+    }
+  }
+
+  it("Renders expected fields", () => {
+    const component = mount(<FeaturedCollectionsRails {...props} />)
+
+    expect(component.text()).toMatch("Featured Collections")
+    expect(component.text()).toMatch("Art Inspired by Cartoons")
+    expect(component.text()).toMatch("Street Art: Celebrity Portraits")
+    expect(component.text()).toMatch("Street Art: Superheroes and Villains")
+  })
+
+  it("Renders no arrows when there are less than 5 collections", () => {
```

Comment:
So this is pedantic, but it's always rendering the left arrow if I understand the code correctly, right? Can we rename this test? Saying "no arrows" in the test description but asserting that one arrow exists feels slightly cognitive dissonance-y.